src/Api/Api.csproj

```diff
@@ -5,6 +5,7 @@
     <MvcRazorCompileOnPublish>false</MvcRazorCompileOnPublish>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <ANCMPreConfiguredForIIS>true</ANCMPreConfiguredForIIS>
+    <IncludeSourceRevisionInInformationalVersion>false</IncludeSourceRevisionInInformationalVersion>
```

Comment:
No, currently on master `CoreHelpers.GetVersion()` only returns the version, like: `2022.6.2`. From [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props):

> If the $(SourceRevisionId) property is present, it's appended to InformationalVersion. You can disable this behavior using IncludeSourceRevisionInInformationalVersion.

In order to keep existing behavior the same, I added the `IncludeSourceRevisionInInformationalVersion` and set it to `false` so that we can capture the `gitHash` independently. By default `IncludeSourceRevisionInInformationalVersion` is true, so if we omit it then our version will become `{version}+{gitHash}`. We could go for this approach and `split()` on the `+`, but I wanted to keep behavior as similar as possible. Let me know if we should change the approach.

Reply:
Ah good to know. I wasn't aware that adding `SourceRevisionId` changed the behavior. We should add this to the `Directory.Build.props`.

Reply:
Yes, I agree! Updated.